approval_polls/templates/base.html

```diff
@@ -45,6 +45,11 @@
               </a>
             </div>
           </div>
+          <div class='row-fluid'>
+            <div class='col-md-8 col-md-offset-4'>
+              <a href='{% url 'approval_polls:tag_cloud' %}'><p width='100%'>View trending polls!</p></a>
```

Comment:
Since there is no validation currently for tags, adding a random tag eg. new-york. And then clicking on the link results in an error - 
```
Exception Type: | NoReverseMatch
Reverse for 'tagged_polls' with arguments '(u'test-test',)' and keyword arguments '{}' not found. 1 pattern(s) tried: [u'approval_polls/tag/(?P<tag>[\\s\\w]+)/$']
/Users/khilarr/dev/venv/lib/python2.7/site-packages/django/core/urlresolvers.py in _reverse_with_prefix, line 496
```
Could you validate to only allow space and hypen apart from alphanumeric characters for tag inputs ? That way I think the reverse match would also be easier to account for.